samples/caf/src/modules/led_state.c

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr.h>
+
+#define MODULE led_state
+#include <caf/events/module_state_event.h>
+#include <caf/events/led_event.h>
+#include <caf/events/button_event.h>
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(MODULE, CONFIG_LOG_DEFAULT_LEVEL);
+
+#include "led_state_def.h"
+
+
+enum button_id {
+	BUTTON_ID_NEXT_EFFECT,
+	BUTTON_ID_NEXT_LED,
+
+	BUTTON_ID_COUNT
+};
+
+static enum led_effect_id curr_led_effect_id[LED_ID_COUNT];
+static enum led_id curr_led_id;
+
+
+static void send_led_event(enum led_id led_id, const struct led_effect *led_effect)
+{
+	__ASSERT_NO_MSG(led_effect);
+	__ASSERT_NO_MSG(led_id < LED_ID_COUNT);
+
+	struct led_event *event = new_led_event();
+
+	event->led_id = led_id;
+	event->led_effect = led_effect;
+	EVENT_SUBMIT(event);
+}
+
+static bool handle_button_event(const struct button_event *evt)
+{
+	ARG_UNUSED(evt);
```

Comment:
ARG_UNUSED ?

Reply:
Uhmm. Leftover. I wonder why the app is working with that :D I will remove this ofc.